manifests/init.pp

```diff
@@ -220,17 +214,17 @@
   }
 
   if ($proxy_enabled or $host != $sentry::params::host) and
-      $password_hash == $sentry::params::password_hash {
-    notify { 'Password hash unchanged from default, this is a security risk!': }
+      $password == $sentry::params::password {
+    notify { 'Password unchanged from default, this is a security risk!': }
   }
   if ($proxy_enabled or $host != $sentry::params::host) and
       $secret_key == $sentry::params::secret_key {
     notify { 'Secret key unchanged from default, this is a security risk!': }
   }
   if $version and (
-      versioncmp($version, '7.0.0') < 0 or versioncmp($version, '8.0.0') >= 0
+      versioncmp($version, $sentry::params::version) < 0 or versioncmp($version, '8.0.0') >= 0
```

Comment:
I would suggest calling `$sentry::version` here because if `$sentry::params::version` is overwritten it wont take effect here.